mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -2318,8 +2319,8 @@ class LogisticRegressionSuite
       assert(m1.interceptVector ~== m2.interceptVector absTol 0.05)
     }
     val testParams = Seq(
-      ("binomial", smallBinaryDataset, 2),
-      ("multinomial", smallMultinomialDataset, 3)
+      ("Binomial", smallBinaryDataset, 2),
```

Comment:
What about "binomial" and "BiNoMiaL"? Also, what about doing:

````scala
lr.setFamily("BiNomial")
assert(lr.getFamily === "binomial")
````

Also, I'm not a big fan of sprinkling these very subtle "tests" around the test suite. We should have dedicated tests of this functionality. Otherwise, how should future developers know that the capital "B" here is supposed to be there and is actually testing some desired functionality?